ccd-definition/AuthorisationCaseEvent/systemUpdate.json

```diff
@@ -12,5 +12,47 @@
     "CaseEventID": "NOTIFY_DEFENDANT_SOLICITOR_FOR_CLAIM_ISSUE",
     "UserRole": "caseworker-civil-systemupdate",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "NOTIFY_CLAIMANT_SOLICITOR_FOR_REQUEST_FOR_EXTENSION",
```

Comment:
Might be worth looping in Alison for these event names, I don't know if they will need to be specific to each solicitor for multiparty

Reply:
@pd-dudkiewicz as discussed, for this event it can be NOTIFY_APPLICANT_SOLICITOR1_FOR_REQUEST_FOR_EXTENSION. If you need to underscore before the integer then go for it. We need to review all the other events to future proofing.